runtime_lib/CMakeLists.txt

```diff
@@ -20,6 +20,10 @@ set(AIE_RUNTIME_TEST_TARGET_VAL ${AIR_RUNTIME_TEST_TARGET})
 if(NOT x86_64_TOOLCHAIN_FILE)
   set(x86_64_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/../cmake/toolchainFiles/toolchain_x86_64.cmake")
 endif()
+# For now, using the same toolchain file for x86_64 and HSA, can seperate if needed
+if(NOT x86_64-hsa_TOOLCHAIN_FILE)
+  set(x86_64-hsa_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/../cmake/toolchainFiles/toolchain_x86_64.cmake")
+endif()
```

Comment:
there's nothing in that toolchain file https://github.com/Xilinx/mlir-aie/blob/7fa04a7654bb3be1420bb8ad31e345bc329fc041/cmake/toolchainFiles/toolchain_x86_64.cmake#L6

Reply:
Just want x86_64-hsa to use the same as x86_64. My thought process was this is more of a placeholder but totally happy to delete it if we have no need for the toolchain file.

Reply:
I don't think the BUILD_AIR_PCIE in that toolchain file is referenced anywhere.

Reply:
Removed the unused BUILD_AIR_PCIE